TitanRobotics2024/src/main/java/frc/robot/Subsystem/IntakePivot.java

```diff
@@ -0,0 +1,149 @@
+package frc.robot.Subsystem;
+
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.robot.Data.ButtonMap;
+import frc.robot.Data.PortMap;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.trajectory.TrapezoidProfile.State;
+import frc.robot.Subsystem.ModifiedMotors;
+import frc.robot.Subsystem.ModifiedEncoders;
+
+public class IntakePivot implements Subsystem
+{
+    private static IntakePivot instance = null;
+    private ModifiedMotors pivotMotor;
+    private ModifiedEncoders pivotEncoder;
+    private ProfiledPIDController pivotProfiledPIDController;
+    private double kP = 0.1;
+    private double kI = 0.0;
+    private double kD = 0.0;
+    private double kSVolts = 0.0;
+    private double kGVolts = 0.0;
+    private double kVVolts = 0.0;
+    private double kAVolts = 0.0;
+    private double currentPosition;
+
+    private double maxVelocity;
+    private double maxAcceleration;
+    private double encoderDistancePerRotation = 360.0;
+    private double encoderPositionOffset = 0.0;
```

Comment:
```suggestion
```
unused? or should it be the 120 on line 143

Reply:
reject